.github/workflows/docs.yml

```diff
@@ -31,10 +31,10 @@ jobs:
     - name: Checkout android-maps-ktx
       uses: actions/checkout@v3
 
-    - name: Set up JDK 17
+    - name: Set up JDK
       uses: actions/setup-java@v3
       with:
-        java-version: '17'
+        java-version: '21'
```

Comment:
As a bonus, you should be able to use latest version to compile the project, there is no reason to stick to an old java for the build 🤷